src/block-components/separator/style.scss

```diff
@@ -5,8 +5,8 @@
 		.stk-inner-blocks {
 			z-index: 5;
 		}
-		> *:not(.stk-video-background) { // Don't change the order of any video backgrounds, they stay at the back.
-			z-index: 2;
+		> *:not(.stk-video-background, .stk-inner-blocks) { // Don't change the order of any video backgrounds, they stay at the back.
+			z-index: -1;
```

Comment:
I'd recommend doing something else here because this currently changes the z-index of all existing content. The fix should only affect the broken parts.